src/main.py

```diff
@@ -0,0 +1,8 @@
+import sys
+from PyQt5.QtWidgets import QApplication
+from MainWindow import MainWindow
+
+if __name__ == '__main__':
+    app = QApplication(sys.argv)
```

Comment:
Все круто, но можешь еще подрубить кастомные шрифты, дефолтные параша и отображаются как-то криво местами.
```suggestion
    app = QApplication(sys.argv)
    QFontDatabase.addApplicationFont(os.path.join(Environment.Resources.path(), "fonts", "Roboto", "Roboto-Bold.ttf"))
```